source/main/modules/window-manager/index.ts

```diff
@@ -121,7 +127,17 @@ export default class WindowManager {
           callingWindow.minimize()
           break
         case 'win-close':
-          callingWindow.close()
+          if (process.platform !== 'darwin') {
+            const showInNotification = Boolean(global.config.get('system.showInNotification'))
+            if (showInNotification) {
+              callingWindow.hide()
+            }
```

Comment:
Fails `yarn lint`: `134:13  error    Closing curly brace does not appear on the same line as the subsequent block`